### DISCUSSION TITLE
`0.peerjs.com` blocks non-browser clients?

### DISCUSSION THREAD

**Discussion by WofWca** [peers/peerjs#1259] [Q&A] [answered]
I'm trying to use https://github.com/muka/peerjs-go , but I get 403 from Cloudflare when an attempt to create a `wss` connection is made. However, connecting to the same URL through a browser works alright. I've created an issue about this https://github.com/muka/peerjs-go/issues/19.

I guess the easiest way to reproduce would be to `curl -v 0.peerjs.com`, though I know it's not WSS, but it gives the same 403, unlike if you go to that URL using a browser.

Is this done on purpose? Is there anything I can do to fix this? Is there anything the server maintainer can do to fix this?

**Comment by jonasgloning** [comment]
I recently had to increase the 'Security Level' on Cloudflare because the server was being flooded by millions of connections per hour. It's likely that peerjs-go clients are blocked because they are detected as not originating from a browser.

This measure should be temporary — but I have to prioritise uptime for the majority of users.

I've just lowered the setting again. Please let me know if it works for you now. If there's demand and someone has an idea for distribution, we could also issue API keys for non-browser clients that would allow them to bypass the WAF.

**Reply by WofWca** [reply, marked as answer]
Thanks! I just tested it and it works.
Don't feel pressured though because I personally have the resources to set up my own PeerJS server.
